zips/zip-0228.rst

```diff
@@ -2,12 +2,539 @@
+|``varies``                   |``SwapActionGroup``       |``ActionGroupDescription``                 |The description of the Action Group corresponding to the desired     |  
+|                             |                          |                                           |Swap operation. It must contain at least two Actions, the Swap       |  
+|                             |                          |                                           |Action and the Fee Action.                                           |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+
+The ``SwapIO`` data type is used to represent the input and output of a Swap Order.
+
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+| Bytes                       | Name                     | Data Type                                 | Description                                                         |
++=============================+==========================+===========================================+=====================================================================+
+|``32``                       |``asset_base``            |``byte[32]``                               |The Asset Base [#zip-0227-assetbase]_ of the input/output of the     |  
+|                             |                          |                                           |Swap Order.                                                          |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+|``8``                        |``qty``                   |``uint64``                                 |The quantity of the Asset represented by ``asset_base`` proposed in  |  
+|                             |                          |                                           |the Swap Order.                                                      |  
++-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
+
+Rationale for Swap Orders
+-------------------------
+
+In a Swap Order, senders only manipulate their funds. 
+This means that senders of Orders spend their notes (input) denominated in an Asset :math:`\mathsf{A}_1` and create a note for themself, to receive the desired amount of another Asset :math:`\mathsf{A}_2`.
+
+To support a Closed Ledger Order Book (CLOB)-like trading environment, the Order senders do not know in advance with whom their orders will be matched, so they do not know, in advance, who is the recipient of the funds they offer to swap. 
```

Comment:
It is more fundamental then CLOB. Party B or a matcher need to see what is the content inside. Avoid focusing on CLOBS. focus on the simple B2B case and mention CLOBs/matchers when needed.

Reply:
Made some edits to remove the CLOB highlight